CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Fixed an issue where lane guidance icons would indicate the wrong arrow for certain maneuvers. ([#2796](https://github.com/mapbox/mapbox-navigation-ios/pull/2796), [#2809](https://github.com/mapbox/mapbox-navigation-ios/pull/2809) 
 * Fixed a crash showing a junction view. ([#2805](https://github.com/mapbox/mapbox-navigation-ios/pull/2805))
 * Fixed an issue with CarPlay visual instructions where U-Turn maneuver icons were not being flipped properly based on regional driving side ([#2803](https://github.com/mapbox/mapbox-navigation-ios/pull/2803)) 
+* Add support for iOS 13's UIScene based CarPlay API ([#2832](https://github.com/mapbox/mapbox-navigation-ios/pull/2832))
```

Comment:
What does this mean in terms of the developer’s migration path? Should we say something like this?

```suggestion
* Added support for the `UISceneDelegateClassName` Info.plist key for integrating `CarPlayManager` with `UIScene` on iOS 13 and above. ([#2832](https://github.com/mapbox/mapbox-navigation-ios/pull/2832))
```